### PR TITLE
(doc) Add notes on Docker container

### PR DESCRIPTION
**Pull request by pauby**
## Description Of Changes
Add notes using Docker container, to preview the site, with the VS Code extension and without.

## Motivation and Context
There were no notes.

## Testing

~~* [ ] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.~~

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left-hand side)/
* [ ] Menu structure has been updated

## Related Issue
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,29 +19,56 @@ Listed below are some of the areas we consider important when writing. We have t
...
 
-### Building the site
+* Using the Visual Studio Code Dev Containers extension.
+* Running the Docker container from the command line.
 
-To build the site locally on your machine, either run the `.\build.ps1` or the `build.sh` file (depending on your system).  This will compile the site, and all generated output file be placed into the `output` folder.
+#### Using the Visual Studio Code Dev Containers Extension
 
-### Previewing the site
+Connect to the Docker Container in Visual Studio Code. This will launch Visual Studio Code and open a terminal that is running inside the Docker Container whose configuration is defined in `/.devcontainer/devcontainer.json` and `/.devcontainer/Dockerfile`. You can now move onto [previewing the site](#previewing-the-site).
```
Comment (JPRuskin) [inline README.md:51]: Is it not simpler to open the repository in VSCode _as a Docker container_? That handles all of the rest of it, including getting the right image, starting the container, and connecting to it ready for usage.

image

Comment (pauby) [reply README.md:51]: That's what I meant with 'Connect to the Docker Container in Visual Studio Code'. The rest of it was just to make it clear where the config was stored.

Comment (JPRuskin) [reply README.md:51]: Ah, sorry. I read the line as written (slash the doc as written, when applied to this section) to be "stand up the Docker container manually, and then use `Attach to running container`" (which is a different specific option).

**Comment by gep13** [comment]
@pauby thanks for getting this added!  I have enabled auto-merging of this PR.

**Pull request merged**